chore: Mark screenshot option as experimental (#375)
getsentry/sentry-godot: CHANGELOG.md

# Changelog

> ## Unreleased
> 
> ### Breaking changes
> 
> - The `attach_screenshot` and `screenshot_level` options have moved to the experimental section. If you previously had it enabled, you will need to re-enable it in its new location.
> 
> ### Other changes
> 
> - Marked screenshot option as experimental and not recommended for production use ([#375](https://github.com/getsentry/sentry-godot/pull/375))
>   - Note: we plan to improve it.

Reviewer: nit: IDK what the convention is in this repo but elsewhere I've seen simple breaking change, it would be just a single changelog item

```suggestion
## Unreleased

### Breaking changes

- The `attach_screenshot` and `screenshot_level` options have moved to the experimental section while we're still improving things. If you previously had it enabled, you will need to re-enable it in its new location. They're currently not recommended for production use. ([#375](https://github.com/getsentry/sentry-godot/pull/375))

```
